File: 5KYU/RGB to Hex Conversion/solution.cs
public class Kata
{
	public static string Rgb(int r, int g, int b) 
	{
		r = Clamp(0, 255, r);
		g = Clamp(0, 255, g);
		b = Clamp(0, 255, b);
    
		return r.ToString("X2") + g.ToString("X2") + b.ToString("X2");
	}
	public static int Clamp(int min, int max, int val)
		=> val < min ? min : val > max ? max :val;
}

File: 5KYU/RGB to Hex Conversion/solution.clj
(ns rgb.core
  (:require [clojure.string :as str]))

(defn clamp [min max v]
  (cond (< v min) min
        (> v max) max
        :else v))

(defn rgb [r g b]
  (let [rng (partial clamp 0 255)
        r (rng r)
        g (rng g)
        b (rng b)]
    (str/upper-case (str (format "%02x" r) (format "%02x" g) (format "%02X" b)))))